meta/labs/README.md

```diff
@@ -3,6 +3,24 @@
 This document will contain a brief overview of our labs program and what it'll
 entail going into the 2015 fall semester.
 
+## Format
+
+The Labs program will follow this format:
+
+1. Workshops
+   - These are based on the workshops from Hack Camp
```

Comment:
This top section (not including the case studies) I don't think accurately captures labs.